sdk/healthinsights/health-insights-radiologyinsights-rest/review/health-insights-radiologyinsights.api.md

```diff
@@ -42,6 +46,15 @@ export type AzureHealthInsightsClient = Client & {
     path: Routes;
 };
 
+// @public (undocumented)
```

Comment:
Can we document this or exclude it from the public API? I am not an expert in thepackage but this function looks like an internal implementation detail, or do we expect customers to use this?

Reply:
@maorleger is this blocking the release or can we release with this included?

Reply:
Well, public API changes are breaking post-GA. So if we are going to GA this (which I believe is the case but let me know if I misunderstood) then we should address this: either by documenting the function if it's meant to be part of the public API or removing it (by not exporting it at the top level).